README.md

```diff
@@ -193,6 +194,14 @@ Example:
 $ python sumo_mgmt.py -url https://api.sumologic.com/api/v1/ -accessid [YOUR ACCESS ID] -accesskey [YOUR ACCESS KEY] -deleteOfflineCollectors [aliveBeforeDays]
 ```
 
+### Assign Ingest Budget to Collectors
+
+A filtered set of collectors can be assigned to an Ingest Budget by using the -assignBudget command with the ID of the budget you'd like to assign.
```

Comment:
we should link "Ingest Budget" to the help docs (I did in the previous comment I left)

Reply:
should also add backticks for `-assignBudget` command

Reply:
backticks are already there for example command